src/components/Slogans/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { clsx } from 'clsx'
+import { montserrat } from '@/styles/fonts'
+import { t } from 'i18next'
+
+const SlogansData: string[] = t('slogans', { returnObjects: true })
+
+const Slogans: JSX.Element[] = SlogansData.map((slogan, index) => {
```

Comment:
TypeError: Cannot read properties of undefined (reading 'map')

hatası alıyorum buradan

Reply:
Evet öngörülen hata olarak not geçecektim unutmuşum , hatırlattığın için teşekkürler . Çalışmayı engellemeyen bir hata , i18n yapısı şuan SSR için uygun yapılandırılmadığından anlık olarak atıyor , language yapılandırılması ays-336 ile ele alınacak olduğundan bu issue dahilinde çözüme kavuşturacağım. Senin de eline sağlık.

Reply:
bir sonraki ticket/issue ve merge ile çözülecek olsa da bence bu PR'ı main'e runtime errorlu mergelemeyelim. ama son kararı sana bırakıyorum, merge'ü blocklamaması için resolveluyorum bunu.

Reply:
Yani çok bi etkisi olmayacak ama , i18 yerine direkt datayı verip sonraki task da eski haline getirebilirim , şuan hata gözükmemesi için.